pkg/kubelet/kubelet.go

```diff
@@ -2217,24 +2207,21 @@ func (kl *Kubelet) recordNodeStatusEvent(event string) {
 var oldNodeUnschedulable bool
 
 func (kl *Kubelet) syncNetworkStatus() {
-	kl.networkConfigMutex.Lock()
-	defer kl.networkConfigMutex.Unlock()
-
-	networkConfigured := true
+	var err error
 	if kl.configureCBR0 {
 		if err := ensureIPTablesMasqRule(); err != nil {
-			networkConfigured = false
-			glog.Errorf("Error on adding ip table rules: %v", err)
+			err = fmt.Errorf("Error on adding ip table rules: %v", err)
+			glog.Error(err)
 		}
 		if len(kl.podCIDR) == 0 {
```

Comment:
`kl.podCIDR`, IIUC, is guarded by `kl.networkConfigMutex`. Remove the lock acquiring here may cause data race.